### ISSUE TITLE
can't install plugins in builder container

### ISSUE THREAD

**Issue by juliomsilva** [Kong/docker-kong#374]
It doesn't seem to be possible to build the custom docker image with the latest version that brings the error below, but works fine using builder with image 0.14.1.

```
luarocks install --tree=system kong-jwt-claims-headers  --only-server=http://localhost:8080
[packer exec] luarocks install --tree=system kong-jwt-claims-headers  --only-server=http://localhost:8080

Error: LuaRocks 3.3.1 bug (please report at https://github.com/luarocks/luarocks/issues).
Arch.: linux-x86_64
/usr/local/share/lua/5.1/luarocks/fs/lua.lua:910: attempt to concatenate local 'downloader' (a nil value)
stack traceback:
	/usr/local/share/lua/5.1/luarocks/fetch.lua:69: in function 'fetch_url'
	/usr/local/share/lua/5.1/luarocks/fetch.lua:33: in function 'fetch_caching'
	/usr/local/share/lua/5.1/luarocks/manif.lua:108: in function 'load_manifest'
	/usr/local/share/lua/5.1/luarocks/search.lua:111: in function 'remote_manifest_search'
	/usr/local/share/lua/5.1/luarocks/search.lua:155: in function 'search_repos'
	/usr/local/share/lua/5.1/luarocks/search.lua:241: in function 'find_suitable_rock'
	/usr/local/share/lua/5.1/luarocks/search.lua:263: in function 'find_rock_checking_lua_versions'
	/usr/local/share/lua/5.1/luarocks/cmd/install.lua:239: in function </usr/local/share/lua/5.1/luarocks/cmd/install.lua:215>
	[C]: in function 'xpcall'
	/usr/local/share/lua/5.1/luarocks/cmd.lua:620: in function 'run_command'
	/usr/local/bin/luarocks:38: in main chunk
	[C]: at 0x55cfd1f91310
failed installing rock: 'luarocks install --tree=system kong-jwt-claims-headers  --only-server=http://localhost:8080 ' failed
ERROR: Service 'kong' failed to build: The command '/bin/sh -c /usr/local/openresty/luajit/bin/luajit /packer.lua -- "$INJECTED_PLUGINS"' returned a non-zero code: 1

```

**Comment by TVenuMadhav** [comment]
Is this resolved? because I too have the same problem with  LuaRocks 3.3.1

**Comment by juliomsilva** [comment]
@TVenuMadhav it's not, but I could make it work with Kong 1.5.1
check this repo: https://github.com/juliomsilva/kong-container-with-custom-plugins

**Comment by Tieske** [comment]
@hishamhm this seems to be a LuaRocks bug, from the first looks of it only when using the `--only-server` flag

**Comment by Tieske** [comment]
The problem is actually that and "downloader" for LuaRocks is missing, so adding `wget` solves this.

Fixed in #411
